src/job.ts

```diff
@@ -56,10 +68,15 @@ export class CronJob<OC extends CronOnCompleteCommand | null = null, C = null> {
 		context?: CronJobParams<OC, C>['context'],
 		runOnInit?: CronJobParams<OC, C>['runOnInit'],
 		utcOffset?: CronJobParams<OC, C>['utcOffset'],
-		unrefTimeout?: CronJobParams<OC, C>['unrefTimeout']
+		unrefTimeout?: CronJobParams<OC, C>['unrefTimeout'],
+		errorHandler?: CronJobParams<OC, C>['errorHandler']
 	) {
 		this.context = (context ?? this) as CronContext<C>;
 
+		if (errorHandler != null) {
```

Comment:
once the `null` type is added (as specified by another comment), this null check will be superfluous and can be removed. directly assign the param to the property.

Reply:
Removed all superfluous null checks